components/nimbus/src/lib.rs

```diff
@@ -189,9 +202,9 @@ impl NimbusClient {
         Ok(result)
     }
 
+    // TODO: Remove this, it's not used by consumers
     pub fn update_experiments(&self) -> Result<Vec<EnrollmentChangeEvent>> {
```

Comment:
I'll bring this back, totally unrelated to the change, removed it because I was trying something out: